src/logo/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+export const getGoslingLogoSvg = (width = 20, height = 20) => (
```

Comment:
This function returns JSX, so it requires React to use it: it is a React component. Rather than `getGoslingLogoSvg` why not export `GoslingLogo`:

```tsx
type GoslingLogoProps = {
  width: number;
  height: number;
}

export function GoslingLogo({ width, height }: GoslingLogoProps) {
  return <svg>...</svg>
}
```

I'd also include some JSDoc comments for `GoslingLogoProps` to explain what width and height are.

Reply:
If width and height should always be then same, then I'd go for just `size`.

Reply:
```jsx
import react from 'React';
import { GoslingLogo } from 'gosling.js';

function App() {
  return <GoslingLogo size={20} />
}
```